packages/datadog-plugin-winston/src/index.js

```diff
@@ -3,6 +3,11 @@
 const LogPlugin = require('../../dd-trace/src/plugins/log_plugin')
 
 class WinstonPlugin extends LogPlugin {
+  constructor (...args) {
+    super(...args)
+    this.structured = true
```

Comment:
Let's use a static getter instead as this is really a property of the class and not the instance (we can't use a static property as that's not supported by Webpack 4).

Reply:
Actually at that point we might as well just use a new subclass that overrides `configure` instead so this property is not needed.

Reply:
Can we actually pass through structured as additional part of the args? I believe so?

Reply:
A log plugin class is either structured or not, it cannot be configured one way or the other so an argument doesn't quite make sense for this IMO.

Reply:
Winston actually supports both :D
But our plugin can't handle that

Reply:
Does it though? I thought the end result of our investigation was that it's always structured internally?